NER_Data_collection_and_annotation.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<p><code>elements</code> can be moved as a global variable to reduce duplication.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meetnaren/ner-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Will do.</p>